syncano/models/manager.py

```diff
@@ -645,6 +645,42 @@ def filter(self, **kwargs):
         self.endpoint = 'list'
         return self
 
+    @clone
+    def fields(self, *args):
+        """
+        Special method just for data object :class:`~syncano.models.base.Object` model.
+
+        Usage::
+
+            objects = Object.please.list('instance-name', 'class-name').fields('name', 'id')
+        """
+        self.query['fields'] = ','.join(args)
```

Comment:
we should have some kind of dummy validation here if someone pass invalid or non existing field